data/blog/rene-pickhardt-receives-lts-grant.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: "Long-Term Support For René Pickhardt"
+date: '2024-01-18'
+tags: ['OpenSats', 'grants', 'bitcoin', 'LTS']
+draft: false
+authors: ['cfunk']
+images: ['/static/images/blog/22-rene.jpg']
+summary: "OpenSats Welcomes René Pickhardt as an LTS Grantee."
+---
+
+
+We are very excited to award a Long-term Support Grant to René Pickhardt,
+helping him continue his fundamental research on improving Bitcoin’s overall
+usefulness as a payment system through the Lightning Network.
+
+Fueled by gratitude for his scholarship from the [German National Academic
+Foundation](https://en.wikipedia.org/wiki/Studienstiftung), and the everyday
+benefits of open-source software he uses, René has centered his career on giving
+back to the communities that supported him on his path.
+
+He has consistently shared commercially significant knowledge for a long time
+without seeking prior funding. All major LN implementations have integrated some
+of his research results and algorithms, now widely recognized as the most
+optimal for bitcoin payments.
+
+Over the years, he has achieved an almost endless list of invaluable
+contributions, including creating [Pickhardt
+Payments](https://github.com/renepickhardt/pickhardtpayments), renepay, actively
+answering most questions about the LN on [Stack
+Exchange](https://bitcoin.stackexchange.com/users/81940/rene-pickhardt),
+co-authoring the openly licensed book "[Mastering the Lightning
+Network](https://github.com/lnbook/lnbook)," proposing [Just in Time
+Routing](https://lists.linuxfoundation.org/pipermail/lightning-dev/2019-March/001891.html),
+mentoring at [Chaincode Labs Lightning
+Residency](https://residency.chaincode.com/), and [Summer of
+Bitcoin](https://www.summerofbitcoin.org/mentors), as well as responsibly
+disclosing a blackmail attack related to channel jamming (don’t worry, he didn’t
+have to blackmail us for this glowing review!)
+
+This LTS grant is now instrumental in René's ongoing efforts to enhance reliable
+payment planning, efficient payment routing, and explore innovative liquidity
+management strategies on the Lightning Network. He’s currently focused on
+researching non-minimum cost flow-based methods to optimize the expected
```

Comment:
maybe one could also link this open stack exchange question: https://bitcoin.stackexchange.com/questions/117447/how-to-compute-the-expected-number-of-sats-to-arrive-in-a-probabilistic-payment (as it is still open)

Reply:
He’s currently focused on researching non-minimum cost flow based methods to optimize the expected delivery of Satoshis and is inviting feedback on this question about [how to compute the expected number of sats to arrive in a probabilistic payment flow](https://bitcoin.stackexchange.com/questions/117447/how-to-compute-the-expected-number-of-sats-to-arrive-in-a-probabilistic-payment)